Podcast/SearchPeopleTableViewCell.swift

```diff
@@ -79,7 +79,11 @@ class SearchPeopleTableViewCell: UITableViewCell {
     
     func configure(for user: User, index: Int) {
         self.index = index
-        profilePictureImageView.image = #imageLiteral(resourceName: "sample_profile_pic")
+        if let url = user.imageURL, let data = try? Data(contentsOf: url) {
+            profilePictureImageView.image = UIImage(data: data)
+        } else {
+            profilePictureImageView.image = #imageLiteral(resourceName: "sample_profile_pic")
```

Comment:
Also why are you not doing UIImage(name: "sample_profile_pic")? If there is not a good reason then switch to that